bandprotocol/cosmoscan
# feat: prepare_gas_used and execute_gas_used

### Issue: (Jira issue Number or URL)
https://bandprotocol.atlassian.net/jira/software/projects/FS/boards/13?selectedIssue=FS-928

### What is the feature?
Show `prepare_gas_used` and `execute_gas_used` in request page

### What is the solution?
add `prepare_gas_used` and `execute_gas_used` in RequestSub and the request page

### Type of change


### How to test?
1. Go to the Request page and select one
2. It should show `prepareGasUsed/prepareGas` and `executeGasUsed/executeGas`

### Screenshots (if any)
<img width="1221" alt="Screenshot 2566-04-25 at 16 53 29" src="https://user-images.githubusercontent.com/12908129/234241550-1fcbb345-170d-4550-bf74-888569b47ece.png">


### Other Notes
Add any additional information that would be useful to the developer or QA tester
